source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+- `Run App Store versions of the Mattermost mobile apps <https://docs.mattermost.com/mobile/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps>`_
+
+----
+
+#############################
+Appendix B: EMM Provider List
+#############################
+
+We’ve compiled a list of the most popular EMM providers and the associated Unified Endpoint Management tools they employ. In two cases (Blackberry Dynamics and MobileIron) we’ve also provide corresponding documentation developed by the Mattermost team.
+
+************************************
+Blackberry Dynamics (Blackberry UEM)
+************************************
+
+“BlackBerry UEM is a multiplatform EMM solution that provides comprehensive device, app, and content management with integrated security and connectivity, and helps you manage iOS, macOS, Android, Windows 10, and BlackBerry 10 devices for your organization. BlackBerry UEM is included in the `BlackBerry Secure UEM & Productivity Suites <https://www.blackberry.com/us/en/products/blackberry-secure-uem-suites.html>`_ - Choice Suite, Freedom Suite, and Limitless Suite.”
+
+- `BlackBerry Website <https://www.blackberry.com/us/en/products/blackberry-uem>`_
+- `BlackBerry Mattermost Documentation <https://docs.mattermost.com/mobile/mobile-blackberry.html>`_
+- `Blackberry Documentation and Help Materials <https://docs.blackberry.com/en/endpoint-management/blackberry-uem/12_11>`_
+
+**********
+MobileIron
+**********
+
+“MobileIron Unified Endpoint Management (UEM) provides the foundation for the industry’s first mobile-centric, zero trust enterprise security framework. Unlike other UEM solutions, MobileIron UEM puts enterprise mobile security at the center of your enterprise and allows you to build upon it with enabling technologies such as `zero sign-on (ZSO) <https://www.mobileiron.com/en/products/access>`_ user and device authentication, multi-factor authentication (MFA), and `mobile threat detection (MTD) <https://www.mobileiron.com/en/products/mobile-threat-defense>`_.”
+
+- `MobileIron Website <https://www.mobileiron.com/en/products/uem>`_
+- `MobileIron Mattermost Documentation <https://docs.mattermost.com/mobile/mobile-mobileiron.html>`_ 
+- `MobileIron Support (requires login) <https://help.mobileiron.com/s/login/?startURL=%2Fs%2F&ec=302>`_
+
+****
+Fyde
+****
+“Securely access Mattermost’s messaging platform to enable productive team collaboration. Our connectionless, modern alternative to VPN helps mitigate breach risk by securing private cloud network from direct access by unmanaged devices.”
+
+- `Fyde for Mattermost Use Case <https://www.fyde.com/use-cases/fyde-for-mattermost>`_
+- `Fyde Documentation <https://fyde.github.io/docs/>`_
+
+****************************************
+Workspace One (formally VMware AirWatch)
```

Comment:
```suggestion
Workspace One (formerly VMware AirWatch)
```
I could be wrong with this change, so please confirm. :)

Reply:
+1. Typo.